### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.7)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29363]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Ryan Thurlwell (OCTO Design Lead)

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
On the map results, the number is inside an oval and the font family is incorrect

### VA.gov Experience Standard
[Cateogy Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
The number should be inside of a circle and change font family to Source Sans

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by af-manish** [comment]
@rtwell  Can you look at this and follow-up?

**Comment by allison0034** [comment]
@af-manish again, sorry for the delay. I looked at the latest Sketch file and the left hand location with the number is now in a circle. Thank you

**Comment by jfinnern** [comment]
Validated in staging

**Comment by ERRonat0609** [comment]
@jfinnern - looking at your test script, I don't know how to determine that the font is the correct one. I need to demo this on Thursday afternoon, so if you could let me know how I show EDU that the font is correct, I would appreciate it. Thanks!

**Comment by jfinnern** [comment]
@ERRonat0609 - Hi Gina, you can do an Inspect Element within the browser that will display the font info. I'll add to the test case for reference.

**Comment by ERRonat0609** [comment]
UAT approved via demo during Digital Engagement meeting on 12/9/21. (Tammy Hurley)
